src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/KeywordCompletionProviderTests.vb

```diff
@@ -173,5 +173,23 @@ End Class
 
             VerifyNoItemsExist(code)
         End Sub
+
+        <WorkItem(4167, "https://github.com/dotnet/roslyn/issues/4167")>
+        <Fact(), Trait(Traits.Feature, Traits.Features.KeywordRecommending)>
+        Public Sub ImplementsAfterSub()
```

Comment:
I support the existence of this test; are there explicit tests for the syntax facts method? Just to avoid spooky-action-at-a-distance tests.